JitsiConference.js

```diff
@@ -389,6 +395,17 @@ JitsiConference.prototype._init = function(options = {}) {
     );
 
     this._signalingLayer.setChatRoom(this.room);
+    this._signalingLayer.on(
+        SignalingEvents.SOURCE_UPDATED,
+        (sourceName, endpointId, muted, videoType) => {
```

Comment:
Why do we get the video type here, but for the media type we need to query it?

Reply:
The media type can be inferred from the source-name using `getMediaTypeFromSourceName` but not the video type.

Reply:
👍